auth/src/api/interfaces/hashing-handler.ts

```diff
@@ -7,4 +7,16 @@ export abstract class HashingHandler {
      * @returns The hashed value.
      */
     public abstract hash(value: string): string;
+
+    /**
+     * Hashes a given value and compares it with a second one (that is already hashed).
+     * If `toCompare` has a length of 152 characters, the first 64 characters were used as `salt`.
+     * Otherwise the unhashed values were compared to each other.
```

Comment:
It is returned false. Please adjust docs. Also the length of `toCompare` is not important here (for an external interface no internals are needed).